src/cmds/versions/create.js

```diff
@@ -67,19 +66,45 @@ module.exports = class CreateVersionCommand {
       );
     }
 
-    if (!fork) {
+    let versionList;
+    if (!opts.fork) {
       versionList = await fetch(`${config.get('host')}/api/v1/version`, {
         method: 'get',
         headers: cleanHeaders(key),
       }).then(res => handleRes(res));
     }
 
-    const versionPrompt = promptOpts.createVersionPrompt(versionList || [{}], {
-      newVersion: version,
-      ...opts,
-    });
+    const questions = [
+      {
+        type: 'select',
+        name: 'fork',
+        message: 'Which version would you like to fork from?',
```

Comment:
Having these questions duped in some commands sucks but I'm comfortable with the tradeoff of having duped questions in two commands than having to maintain a question generation library.